lib/index.tpl.html

```diff
@@ -1,8 +1,6 @@
 <!doctype html>
-<!--
-  Initialize the default language to English to meet a11y requirements,
-  it will be subsequently updated by the setLocale action.
--->
+<!-- Initialize the default language to English to meet a11y requirements. -->
```

Comment:
I have added a hook to strip out single line comments, so this text can be removed at build time.